tests/unit/serializer/test_brat.py

```diff
@@ -0,0 +1,489 @@
+import os
+from collections import defaultdict as dd
+from dataclasses import dataclass
+from typing import TypeVar
+
+from pie_datasets.builders.brat import BratDocument
+from pie_modules.document.processing import SpansViaRelationMerger
+from pytorch_ie import AnnotationLayer
+from pytorch_ie.core import Document
+
+from src.utils import get_pylogger
+
+log = get_pylogger(__name__)
+
+D = TypeVar("D", bound=Document)
+import pytest
+from pytorch_ie.annotations import BinaryRelation, LabeledMultiSpan, LabeledSpan
+from pytorch_ie.core import annotation_field
+from pytorch_ie.documents import TextBasedDocument, TextDocumentWithLabeledSpansAndBinaryRelations
```

Comment:
use documents from `pie_modules`, the annotations from `pytorch_ie` will soon be deprecated